src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -129,13 +128,13 @@ public static EntityPlayer getPlayerFromPermPlayer(@NotNull Permissions.Player p
      * @return The player the player if found or null
      */
     @Nullable
-    public static EntityPlayer getPlayerFromUUID(@Nullable UUID uuid)
+    public static EntityPlayer getPlayerFromUUID(@Nullable UUID uuid, @NotNull World world)

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): world [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)